src/test/java/duke/TaskTest.java

```diff
@@ -0,0 +1,98 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import java.util.ArrayList;
+
+public class TaskTest {
+    private static final String SEPARATOR = System.getProperty("line.separator");
+    private final ByteArrayOutputStream OUT = new ByteArrayOutputStream();
+    private final PrintStream ORIGINAL_OUT = System.out;
+    private TaskList testTasks;
+    
+    @Test
+    public void testGetTasks() {
+        ArrayList<Task> testTasks = new ArrayList<>();
+        testTasks.add(new ToDo("description", false));
+        testTasks.add(new ToDo("done", true));
+        this.testTasks = new TaskList(testTasks);
+        assertEquals(this.testTasks.getTasks(), testTasks);
+    }

```

Comment:
As per iP task on code quality, should this test be documented?

Reply:
Thanks for the commentary. Yes, I think it needs to be documented since it is `public`. Thanks for pointing out.